js/window.js

```diff
@@ -355,6 +357,17 @@ class Window extends EventEmitter {
 		glfw.setWindowSize(this._window, width, height);
 	}
 	
+	get resizable() {
+		const resizable = glfw.getWindowAttrib(this._window, glfw.RESIZABLE);
+		this._resizable = resizable === 1 ? true : false;
```

Comment:
instead of `1`, either `glfw.TRUE` or no comparison at all: `this._resizable = !!resizable`

Reply:
Removed the comparison. Nice catch.